objects/project-objects/screen.md

```diff
@@ -2,37 +2,41 @@
 
 ![](../../.gitbook/assets/iconscreen.png)
 
-The **Screen Object** in **INCARI** contains the technical specifications of the physical display that the **Project** will ultimately displayed on. In addition, it also has **Attributes** relating to the virtual **Camera** and rendering.
+The **Screen Object** in Incari contains the technical specifications of the physical display that is used to render a **Scene**. Also, it has **Attributes** relating to the virtual **Camera**, the **Simulation Window** and post-processing effects during rendering.
+
+Note: **Screen Objects** pass on **Attributes** to [**Scene Objects**](scene.md) that are created from them.  
```

Comment:
I don't really understand this sentence.

Reply:
I removed it